### PR TITLE
vitamin-r: change license to :commercial; add older versions for old OSs

### PR DESCRIPTION
**Pull request by johntrandall**
I'm not sure if I handled the appcast URL correctly.
What is the best way to handle it when the appcast doesn't include older versions in it?

### CODE REVIEW

**Comment by vitorgalvao** [comment]
+ `appcast` should only be present in the most up-to-date one.
+ Double quotes are reserved for cases where string interpolation is needed. Please change other cases to single quotes.
+ There are too many empty lines, the cask is really confusing, please remove empty lines inside each `elsif`.
+ Those comments should be removed.
+ Prior to version `2` the app must’ve have a different name, correct? `app` needs then to be set in each, with the appropriate name.

**Comment by johntrandall** [comment]
changed double quotes to single quotes. 
appcast now only appears once, removed extra entries (and corresponding comments)
removed empty lines as requested. 
corrected app for older, pre V2 cases

File: Casks/vitamin-r.rb [hunk 1]
```diff
@@ -1,12 +1,32 @@
...
+    sha256 'cfc107e016e364ba1ed2ca091b6b03daf00d748852d06a4de7c421b8f4ece175'
+    url 'http://www.publicspace.net/download/Vitamin_2_19.dmg'
+    app 'Vitamin-R 2.app'
+  else
+    version :latest
+    sha256 :no_check
+    url 'http://www.publicspace.net/download/Vitamin.dmg'
+    appcast 'http://www.publicspace.net/app/vitamin2.xml'
+  end
+    name 'Vitamin-R'
```
Comment (vitorgalvao) [inline Casks/vitamin-r.rb]: The empty line above `name` should remain.

File: Casks/vitamin-r.rb [hunk 2]
```diff
@@ -1,12 +1,32 @@
...
+    sha256 'cf5676f710a9be0f290aa20c2b1f9feaf87f162b580987f7c08477a2f13b2705'
+    url 'http://www.publicspace.net/download/Vitamin_1_85.dmg'
+    app 'Vitamin-R.app'
+  elsif MacOS.release <= :lion
+    version '2.06'
+    sha256 'd6a5839f7a1bdf1b3767f1c48881048f6bee20bf9de55be3936bab3202ee2541'
+    url 'http://www.publicspace.net/download/Vitamin_2_06.dmg'
+    app 'Vitamin-R 2.app'
+  elsif MacOS.release <= :mavericks
+    # last version with compatablity for both :mountain_lion and :mavericks
```
Comment (vitorgalvao) [inline Casks/vitamin-r.rb]: This comment should also be removed.

**Comment by vitorgalvao** [comment]
Thank you. Looking good. Only a few more tweaks. After could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by johntrandall** [comment]
rebased

**Comment by vitorgalvao** [comment]
Looks great. Thank you for the contribution and the fast responses.

**Pull request merged**